code/core/src/phone/java/com/adobe/marketing/mobile/services/ui/AEPMessage.java

```diff
@@ -68,7 +68,7 @@ class AEPMessage implements FullscreenMessage {
     // private vars
     private WebView webView;
     private CardView webViewFrame;
-    private ViewGroup.LayoutParams params;
+    private FrameLayout.LayoutParams params;
```

Comment:
CardView extends FrameLayout directly. Using FrameLayout LayoutParams resolves the incorrect placement of messages on Android API 22 and below.